schemas/context/media-timed-asset-reference.example.1.json

```diff
@@ -4,12 +4,6 @@
   "dc:creator": "Video Author",
   "dc:publisher": "tvonline",
   "xmpDM:duration": 87,
-  "xdm:Audio": {
-    "id3:Audio": {
-      "id3:TRSN": "Q991.3",
-      "id3:TPUB": "Atlantic"
-    }
-  },
```

Comment:
Why is this being removed. If a mistake then remove from branch

Reply:
clearly a mistake!